lib/Dialect/Torch/Transforms/DecomposeComplexOps.cpp

```diff
@@ -4015,6 +4042,7 @@ class DecomposeComplexOpsPass
         patterns);
     addPatternIfTargetOpIsIllegal<DecomposeAtenMseLossOp>(patterns);
     addPatternIfTargetOpIsIllegal<DecomposeAtenNormScalarOptDimOp>(patterns);
+    addPatternIfTargetOpIsIllegal<DecomposeAtenRandintOp>(patterns);
```

Comment:
You also need to add the op here: https://github.com/llvm/torch-mlir/blob/07f5f042c787a51716b1b6d18e3999c04720735e/lib/Dialect/Torch/Transforms/LowerToBackendContract.cpp#L363